### PR TITLE
Bandwith delegation supported in the get_account api funcion 

### LINKED ISSUE #978: Add delegate banwith support in the get_account api function

### PR DESCRIPTION
**Pull request by e-schepachev**
Resolve #978.

Additionaly fixed wrong bandwith calculation in the total field and
added staking values to the memory and storage fields in the cleos.

### CODE REVIEW

File: plugins/chain_api_plugin/chain_api_plugin.cpp [hunk 1]
```diff
@@ -181,11 +181,14 @@ namespace {
...
 
-        resource_calculator(chain::resource_limits_manager& rm, chain::account_name name) :
+        resource_calculator(chain::resource_limits_manager& rm, cyberway::chaindb::chaindb_controller& controller, chain::account_name name) :
           rm_(rm),
+          controller_(controller),
           account_(name),
           prices_(rm.get_pricelist()),
           total_stake_(rm.get_account_stake_ratio(fc::time_point::now(), account_, false).numerator),
+          owned_stake_(controller_.find<eosio::chain::stake_agent_object, eosio::chain::stake_agent_object::by_key>
+                       (eosio::chain::stake::agent_key(chain::symbol(CORE_SYMBOL).to_symbol_code(), account_))->balance),
```
Comment (kaynarov) [inline plugins/chain_api_plugin/chain_api_plugin.cpp]: I think instead of **->balance** here should be **->get_total_funds()**

Comment (kaynarov) [reply plugins/chain_api_plugin/chain_api_plugin.cpp]: get_own_funds()

Comment (e-schepachev) [reply plugins/chain_api_plugin/chain_api_plugin.cpp]: here: get_own_funds() - provided

File: plugins/chain_api_plugin/chain_api_plugin.cpp [hunk 2]
```diff
@@ -186,9 +186,10 @@ namespace {
           controller_(controller),
           account_(name),
           prices_(rm.get_pricelist()),
-          total_stake_(rm.get_account_stake_ratio(fc::time_point::now(), account_, false).numerator),
-          owned_stake_(controller_.find<eosio::chain::stake_agent_object, eosio::chain::stake_agent_object::by_key>
-                       (eosio::chain::stake::agent_key(chain::symbol(CORE_SYMBOL).to_symbol_code(), account_))->balance),
+          stake_agent_(controller_.find<eosio::chain::stake_agent_object, eosio::chain::stake_agent_object::by_key>
+                       (eosio::chain::stake::agent_key(chain::symbol(CORE_SYMBOL).to_symbol_code(), account_))),
+          total_stake_(stake_agent_->get_effective_stake()),
```
Comment (afalaleev) [inline plugins/chain_api_plugin/chain_api_plugin.cpp]: in case of `stake_agent_ == nullptr` - here will be segfault.

**Pull request merged**